src/YahooClient.cs

```diff
@@ -5,6 +5,8 @@ public class YahooClient
     private readonly Country Country;
 
     private readonly Language Language;
+    
+    public static bool IsThrottled { get; set; }
```

Comment:
This needs to be a readonly variable

Reply:
I have it as read/write to allow throttling to be selected (or not) on program start. This allows the user to decide which mode to implement.